### DISCUSSION TITLE
v20_loader for Windows

### DISCUSSION THREAD

**Discussion by llelundberg** [aubreymcleod/.v20_format#7] [General]
I have looked at the v20_loader.sh script to see if I could do something similar for Windows.

A cmd script (bat file) has no great way to interact with binary file contet, so I think it would be difficult.

A powershell script would be quite easy, but powershell is way more secure, so then you have to take Executionpolicy and stuff in conciderataion,

A classic VBS file might be the way forward?

**Comment by aubreymcleod** [comment]
That is good call, Visual Basic is a tried and true language, my only question would be whether it might not be better to use a Python script.

 VB hasn't been updated in a while, and truth be told I didn't realize that Windows still natively supported it:D

**Comment by aubreymcleod** [comment]
As an aside, I will drop a link to some test files later tonight, I ran into an issue with my script; things that worked in my IDE were not playing nicely on the outside. So it goes, so I will patch that tonight as well

**Reply by aubreymcleod** [reply]
Found the problem, I use Mac; they moved the default shell to zsh from bash in the last version. So while my script works in bash it does not handle zsh properly.
